apps/backoffice/src/components/headers/__tests__/page-html-head-title..test.tsx

```diff
@@ -0,0 +1,37 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import React from "react";
+import { afterEach, describe, expect, it, vi } from "vitest";
+import { PageHtmlHeadTitle } from "../page-html-head-title";
+
+let section = "";
```

Comment:
what do you think if we added a "test" section in order to verify in the expected that this value is present after the "|" of line 33?